smriprep/workflows/surfaces.py

```diff
@@ -552,9 +567,17 @@ def init_gifti_surface_wf(*, name="gifti_surface_wf"):
                                       ('inflated', 'in3')]),
         (save_midthickness, surface_list, [('out_file', 'in4')]),
         (surface_list, fs2gii, [('out', 'in_file')]),
-        (fs2gii, fix_surfs, [('converted', 'in_file')]),
-        (inputnode, fix_surfs, [('fsnative2t1w_xfm', 'transform_file')]),
-        (fix_surfs, outputnode, [('out_file', 'surfaces')]),
+        (get_surfaces, surfmorph_list, [('thickness', 'in1'),
+                                        ('sulc', 'in2'),
+                                        ('curv', 'in3')]),
+        (get_surfaces, whitesurf_list, [('smoothwm', 'in1'),
+                                        ('smoothwm', 'in2'),
+                                        ('smoothwm', 'in3')]),
+        (whitesurf_list, fscurv2funcgii, [('out', 'in_file')]),
+        (surfmorph_list, fscurv2funcgii, [('out', 'scalarcurv_file')]),
+        (fs2gii, allsurf_list, [('converted', 'in1')]),
+        (fscurv2funcgii, allsurf_list, [('converted', 'in2')]),
+        (allsurf_list, outputnode, [('out', 'surfaces')])
```

Comment:
I think we're going to have trouble conflating `.surf.gii` files and `.shape.gii` files. I think we should probably not combine them and instead make a new `morphometrics` output. Something like:

```suggestion
        (fscurv2funcgii, outputnode, [('converted', 'morphometrics')]),
```

Reply:
that makes a lot of sense -- I'll make sure to do the same moving forward when handling the cifti conversion